package.json

```
@@ -19,7 +19,7 @@
     "generate": "nuxt generate",
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
     "cypress": "cypress",
-    "test:unit": "jest",
+    "test:unit": "cross-env NODE_ENV=test jest",
```

Since you are handling with `Date`, it may be helpful to add a timezone in your tests.

```suggestion
    "test:unit": "TZ='Pacific/Pago_Pago' cross-env NODE_ENV=test jest",
```

You can also set it through Jest... but I don't remember how 😅